en/config_rover/mecanum.md

```diff
@@ -0,0 +1,298 @@
+:::
+
+For [Stabilized mode](../flight_modes_rover/mecanum.md#stabilized-mode) the controller utilizes a closed loop yaw controller, which creates a yaw rate setpoint to control the yaw when it is active:
+
+![Cascaded PID for yaw control](../../assets/airframes/rover/rover_differential/cascaded_pid_for_yaw.png)
+
+Unlike the closed loop yaw rate, this controller has no feed-forward term.
+Therefore you only need to tune the closed loop gains:
+
+1. [RM_YAW_P](#RM_YAW_P) [-]: Proportional gain for the closed loop yaw controller.
+
+   ::: tip
+   In stabilized mode the closed loop yaw control is only active when driving a straight line (no yaw rate input).
+   To tune it set [RM_YAW_I](#RM_YAW_I) to zero and start with a value of 1 for [RM_YAW_P](#RM_YAW_P).
+   Put the rover into stabilized mode and move the left stick of your controller up and/or down to drive forwards/backwards.
+   Disarm the rover and from the flight log plot the _yaw_setpoint_ from the [RoverMecanumSetpoint](../msg_docs/RoverMecanumSetpoint.md) message and the _actual_yaw_ from the [RoverMecanumStatus](../msg_docs/RoverMecanumStatus.md) message over each other.
+   Increase/Decrease the parameter until you are satisfied with the setpoint tracking.
+   :::
+
+2. [RM_YAW_I](#RM_YAW_I) [-]: Integral gain for the closed loop yaw controller.
+
+   ::: tip
+   For the closed loop yaw control an integrator gain is useful because this setpoint is often constant for a while and an integrator eliminates steady state errors that can cause the rover to never reach the setpoint.
+   In [Auto Modes](#auto-modes) there will be a further elaboration on why an integrator is necessary for the yaw controller.
+   :::
+
+The rover is now ready to drive in [Stabilized mode](../flight_modes_rover/mecanum.md#stabilized-mode).
+
+## Position Mode
+
+:::warning
+For this mode to work properly [Acro mode](#acro-mode) and [Stabilized mode](#stabilized-mode) must already be configured!
+:::
+
+[Position mode](../flight_modes_rover/mecanum.md#position-mode) is the most advanced manual mode, utilizing closed loop yaw rate, yaw and speed control and leveraging position estimates.
+
+To configure set the following parameters:
+
+1. [RM_MAX_SPEED](#RM_MAX_SPEED) [m/s]: This is the maximum speed you want to allow for your rover.
+   This will define the stick-to-speed mapping for position mode and set an upper limit for the speed setpoint for all [auto modes](#auto-modes).
+2. [RM_MAX_THR_SPD](#RM_MAX_SPEED) [m/s]: This parameter is used to calculate the feed-forward term of the closed loop speed control which linearly maps desired speeds to normalized motor commands.
+   A good starting point is the observed ground speed when the rover drives at maximum throttle in [Manual mode](../flight_modes_rover/mecanum.md#manual-mode).
+   Increase this parameter if the rover is faster than the setpoint, and decrease if the rover is slower.
```

Comment:
I don't really get what this means - if I measure speed at max throttle, how then do I know what the setpoint is to compare?

I wonder if some of this could be auto-tuned in some way?

Reply:
Updated this in https://github.com/PX4/PX4-user_guide/pull/3411/commits/e9016d6c9799d585384ce3ee6d8bf3a6d76f8254.
Autotuning might be possible, but that is not implemented yet. I'll look into it.

Reply:
Nice. I subedited in https://github.com/PX4/PX4-user_guide/pull/3411/commits/6bd775bd89645a3f42cddeceb98b5ad240110286 

Mostly this indicates my preference for instructions to use lists.